src/table/table.scss

```diff
@@ -4,383 +4,373 @@
+            t(iui-opacity-6)
+          );
         }
+      }
 
-        > .iui-filter.iui-active {
-          visibility: visible;
-  
-          > .iui-icon {
-            @include themed {
-              fill: t(iui-icons-color-primary);
-            }
-          }
-        }
+      > .iui-slot > .iui-more-options {
+        visibility: visible;
+      }
+    }
 
-        &:hover {
-          @include themed {
-            background-color: t(iui-color-background-4);
-          }
+    &:focus-within > .iui-slot > .iui-more-options {
+      visibility: visible;
+    }
 
-          > .iui-filter,
-          > .iui-sort,
-          > .iui-sort > .iui-icon-wrapper {
-            visibility: visible;
-          }
-        }
+    &.iui-row-expanded {
+      @include themed {
+        border-left-color: t(iui-color-background-4);
+        border-right-color: t(iui-color-background-4);
+        border-bottom-color: transparent;
       }
 
-      &.iui-active-sort {
-        @include themed {
-          background-color: t(iui-color-background-4);
-        }
+      .iui-row-expander > .iui-icon {
+        transform: rotate(90deg);
+      }
 
-        > .iui-sort,
-        > .iui-sort > .iui-icon-wrapper {
-          visibility: visible;
+      + .iui-expanded-content {
+        @include themed {
+          border-left-color: t(iui-color-background-4);
+          border-right-color: t(iui-color-background-4);
         }
       }
     }
 
-    &.iui-tables-row-active {
+    // #region Selection
+    &:not(.iui-selected) + .iui-selected,
+    &.iui-selected:first-child {
+      border-bottom-color: transparent;
+    }
+
+    &.iui-selected {
       @include themed {
-        border: solid 1px t(iui-color-foreground-primary);
+        border-color: t(iui-color-foreground-primary);
         background: rgba(t(iui-color-foreground-primary-rgb), t(iui-opacity-5));
       }
-    }
 
-    &.iui-tables-row-new {
-      font-weight: $iui-font-weight-semibold;
+      + .iui-selected {
+        border-bottom-color: transparent;
+        @include themed {
+          border-top-color: rgba(t(iui-color-foreground-primary-rgb), t(iui-opacity-4));
+        }
+      }
 
-      > .iui-tables-filename {
-        position: relative;
+      &:last-child {
+        @include themed {
+          border-bottom-color: t(iui-color-foreground-primary);
+        }
       }
 
-      > .iui-tables-filename::before {
+      + :not(.iui-selected),
+      + .iui-expanded-content + :not(.iui-selected) {
         @include themed {
-          color: t(iui-color-foreground-positive);
+          border-top-color: t(iui-color-foreground-primary);
         }
+      }
+
+      + .iui-expanded-content {
+        @include themed {
+          border-color: transparent t(iui-color-foreground-primary);
+        }
+      }
+    }
+    // #endregion Selection
+
+    // #region Statuses & states
+    &.iui-new {
+      font-weight: $iui-font-weight-semibold;
+      
+      > .iui-main-column::before {
```

Comment:
Do we need different name? It doesn't have any styling only this `::before`. We can just use `iui-cell`

Reply:
How else would you add dot to the right column?